iFixit/react-commerce
# Add SSR support to algolia collection search results

Closes #48 

## Changes

- Renders the collection products server side
- Adds cache control header for caching optimization using stale while revalidate

## QA

1. Visit the Vercel PR preview
2. Visit a collection page and verify that products are already rendered when the page is served
